### ISSUE TITLE
graph.dag() broken

### ISSUE THREAD

**Issue by fonnesbeck** [pymc-devs/pymc3#7]
I am no longer able to generate DAGs for any of 3 models attempted. Fails in the unit test as well. Here is the error reported:

/Users/fonnescj/Code/pymc/pymc/PyMCObjects.pyc in gen_lazy_function(self)
    413 
    414         self._jacobians = {}
    --> 415         for parameter, function in self._jacobian_functions.iteritems():
    416             lazy_jacobian = LazyFunction(fun = function,
    417                                             arguments = self.parents,

```
AttributeError: 'NoneType' object has no attribute 'iteritems'
```

**Comment by jsalvatier** [comment]
Can you send me an example script? I am not familiar with this functionality.

**Comment by fonnesbeck** [comment]
For an arbitrary model, you can generate a pretty DAG, provided that pydot is installed:

M = pymc.MCMC(my_model)
pymc.graph.dag(M)

See the user's guide for details, but there's not much else to it.

**Comment by jsalvatier** [comment]
The issue turns out to be that the numpy deterministics include any() and all() factory functions and graph.py imports using *. I changed to importing using a name, but perhaps it's a bad idea to have pymc deterministics have the same exact names as other functions.

**Comment by jsalvatier** [comment]
fixed by 8c954450da34755a337f
